### DISCUSSION TITLE
We are looking for dotnet/Maui releases a change related to Microsoft.WindowsAppSDK from 1.2.230217.4 to 1.2.230313.1.

### DISCUSSION THREAD

**Discussion by ManbirSinghRakhra** [dotnet/maui#14035] [Q&A] [answered]
Hello,

The following commit regarding WindowsAppSdk is very urgent for us. When are you going to release it?
https://github.com/dotnet/maui/commit/ae7ddf6947466d5c325317b6ca80da8370e47cbb

Will you speed up because MouseFreeze bug in WindowsAppSdk is affecting our Maui Blazor App customers?

Regards and Thanks,
Manbir Rakhra

**Comment by PureWeen** [comment, marked as answer]
@ManbirSinghRakhra for the time being you can add this to your csproj file

```xml
	<ItemGroup>
		<PackageReference Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'" Include="Microsoft.WindowsAppSDK" Version="1.2.230313.1" />
	</ItemGroup>
```

**Reply by ManbirSinghRakhra** [reply]
Thanks very much. This worked 👍
